### PR TITLE
1.0.0rc1

### PR DESCRIPTION
**Pull request by gsheni**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   url: https://pypi.io/packages/source/f/featuretools/featuretools-{{ version }}.tar.gz
-  sha256: f9e65eb19a2fac3aa713ccfcb471f52c99b5b900672784083ac065017cb744ad
+  sha256: befb2b12a2786c244acf83da31dbb512993cebf0b711582ed7a266a3feea62bf
 
 build:
   entry_points:
     - featuretools = featuretools.__main__:cli
   noarch: python
-  number: 2
+  number: 0
```
Comment (gsheni) [inline recipe/meta.yaml:15]: Since the following is true: `if the version changed`

**Review by thehomebrewnerd — APPROVED** [review]

**Review by rwedge — COMMENTED** [review]
We need to update this not to build into main, right?

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge featuretools_rc
```
Comment (gsheni) [inline recipe/conda_build_config.yaml:2]: @rwedge good catch. Added that here

**Comment by gsheni** [comment]
@conda-forge-admin, please rerender

**Pull request merged**